.dockerignore

```diff
@@ -0,0 +1,2 @@
+Dockerfile
+hooks
```

Comment:
Git complains about not having newlines at EOF, I suggest you change your editor settings if you primarily work with git.

Reply:
Fixed both my editor and this file :)